.github/workflows/macos.yaml

```diff
@@ -102,8 +102,9 @@ jobs:
         VERSION: ${{ github.event.inputs.datadog_agent_ref || env.DEFAULT_DATADOG_AGENT_REF }}
       run: |
         bash ./scripts/clone_agent.sh
+        cd $HOME/go/src/github.com/DataDog/datadog-agent
```

Comment:
You might want to change directory before calling `clone_agent.sh` as well. wdyt?